docs/en/latest/FAQ.md

```diff
@@ -35,7 +35,7 @@ In addition, APISIX has dynamic routing and hot loading of plug-ins, which is es
 
 ## What's the performance of APISIX?
 
-One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：[benchmark](https://github.com/apache/apisix/blob/master/benchmark.md)
+One of the goals of APISIX design and development is the highest performance in the industry. Specific test data can be found here：[benchmark](https://github.com/apache/apisix/blob/master/docs/en/latest/benchmark.md)
```

Comment:
We can use `[benchmark](benchmark.md)` directly?

Reply:
yep